packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -52,6 +53,14 @@ const propTypes = {
    * Name of the time input. The name should be unique.
    */
   name: PropTypes.string.isRequired,
+  /**
+   * An identifier used by assistive technologies like screen readers to briefly describe this time input to users.
+   * The label is not rendered visually.
+   *
+   * **BEST PRACTICE FOR ACCESSIBILITY**: you _SHOULD_ set this to match whatever visible label you give in your UI.
+   * Currently this is optional for passivity reasons, but it will become mandatory in a future major version.
+   */
+  label: PropTypes.string,
```

Comment:
This is described in the comment, but in the interest of hand holding our consumers, maybe the name of this prop should be a little more explicit. `a11yLabel` or `visuallyHiddenLabel`, something like that. I could see us ending up with support questions of people trying to add a visual label using this prop thinking it behaved like a form field.

Reply:
Should we also call out that the default label is "Time" if not set?

Reply:
@jmsv6d OK, I will change it to a11yLabel. 
@benbcai  yes, but it must be localized so I am defaulting it to Time further down in the code using formatMessage(). I'll see if I can set the prop default to be the output of a formatMessage(). If not, I will keep it as is.

Reply:
It is not possible to default the a11yLabel (aka label) to a localized string because the locale of the runtime context is not known when that default is assigned to the class. I think what I can do is add something to the constructor taht computes a default and use that to simplify the code base. I'll try it next.

Reply:
It worked! See 89815ff1f9e218cf28a06c70822292773a2cf1ec.